### ISSUE TITLE
GameGuru Max - Add Height / Displacement to PBR (Request)

### ISSUE THREAD

**Issue by nomad5oul** [TheGameCreators/GameGuruRepo#3471]
Please can you add an extra texture slot to PBR so we can use a heightmap or displacement map for any given model / surface.

This is essential to get the visuals up to current generation standards and already supported in Wicked engine.

At the moment surfaces only appear to have depth when looking at them directly and this is limited to normal map which is not ideal.

**Comment by MonkeyFrogStudio** [comment]
Height maps in PBR materials are a standard for game creation. Even when you download materials or purchase them, most will come with a "_h" or "_height" texture so you can include/use it if you decide to. Normal maps add a sense of depth, but can only take you so far. AO helps further by shadowing crevices to help with depth. But both of these methods are only tricks that can fall apart at certain angles. A height map/displacement map for PBR materials allows true 3D where needed. Please do consider adding this and COMPLETING MAX's PBR.

**Comment by Valuky** [comment]
The Implementation of Height / Displacement to PBR is crutial for modern games just like Nomad And Monkey Frog mentioned, With that would be expected a Parallax shader or tesselation, which were on Older MAX's versions but did never work, until it gets removed.

**Comment by bluemeenie195** [comment]
Yeah, after working with custom textures lately, it really is crucial that Height is added for full PBR.

**Comment by MonkeyFrogStudio** [comment]
@bluemeenie195 Height was added to MAX for textures some time ago:

![height](https://github.com/TheGameCreators/GameGuruRepo/assets/36735507/7571d163-7e07-41d4-a7ec-856bbf1c4afd)

**Comment by bluemeenie195** [comment]
sorry, I was thinking this was for terrain textures
